### ISSUE TITLE
Server crash after (last?) player left the server

### ISSUE THREAD

**Issue by DHLF** [cuberite/cuberite#2485]
Because of the weekend I updated the server from build#150 to the latest version.
The server itself worked fine but after leaving the server I could not reenter it because it has crashed.

I tried different client versions first from 1.8.8 down to 1.8."0". No change.

I tried different builds and discovered that build#161 seems to be the latest working version.

```
Info [13:31:28] Player DHLF has joined the game
     [13:31:35] Chunk generator performance: 20.45 ch / sec (41 ch total)
Err  [13:31:35]   D:    | MCServer has encountered an error and needs to close
Err  [13:31:35] Details | SIGSEGV: Segmentation fault
Err  [13:31:35] MCServer Jenkins clang x64 Release (master) build id: #163
Err  [13:31:35] from commit id: 553c695fd2d257191c955db30c377cef4e17c441 built at: Sat 19 Sep 03:37:47 CEST 2015
Err  [13:31:35]   D:    | MCServer has encountered an error and needs to close
Err  [13:31:35] Details | SIGABRT: Server self-terminated due to an internal fault
Err  [13:31:35] MCServer Jenkins clang x64 Release (master) build id: #163
Err  [13:31:35] from commit id: 553c695fd2d257191c955db30c377cef4e17c441 built at: Sat 19 Sep 03:37:47 CEST 2015
```

Linux 3.16.0-4-amd64#1 SMP Debian 3.16.7-ckt11-1+deb8u3 (2015-08-04) x86_64 GNU/Linux

[edit]
@madmaxoft 
I have uploaded the requested information: http://dhlf.de/trash/gdb.txt
And thank you for the interesting hint of command gdb ;)

**Comment by madmaxoft** [comment]
Could you run the problematic build under gdb and capture the stacktrace for the crash?
This forum thread should provide you information on what to do: http://forum.mc-server.org/showthread.php?tid=631
Thanks.

**Comment by bramvdbogaerde** [comment]
I have the same issue.
My stacktrace is here:
https://gist.github.com/bramvdbogaerde/37ec47f84c84c655490e

I'm also running a build from the almost the latest commit (553c695fd2d257191c955db30c377cef4e17c441).

Linux 3.2.34-30 #17 SMP Ubuntu 14.04 armv7l armv7l armv7l GNU/Linux

**Comment by DHLF** [comment]
Tried build#165 and.. that seems to be working, no crash after player left the game

**Comment by tigerw** [comment]
A crash bug related to player deletion was introduced in dd168b0e8b0b814a9e82d7941dbd0137fe09beb9 but rapidly fixed in 49724cd4276785b7a3cfa9450e5df691d3a9de3d. Try the latest build again?

**Comment by bramvdbogaerde** [comment]
It's solved for me now.
